src/components/References/CitationEditor.tsx

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 import AnnotationEdit from '@manuscripts/assets/react/AnnotationEdit'
-import CloseIconDark from '@manuscripts/assets/react/CloseIconDark'
+import TrashIconDark from '@manuscripts/assets/react/TrashIconDark'
```

Comment:
We have `DeleteIcon` here in the style-guide, please check it and see if it's what you need.